packages/video_player/lib/video_player.dart

```diff
@@ -397,6 +405,21 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     value = value.copyWith(volume: volume.clamp(0.0, 1.0));
     await _applyVolume();
   }
+
+  static String _nameOf(VideoFormat format) {
```

Comment:
nits: _formatToString
Could we also add dart unit test for this conversion?

Reply:
Not sure I've ever seen unit tests on private internal methods.